tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Datadog.Trace.ClrProfiler.IntegrationTests.csproj

```diff
@@ -27,6 +27,8 @@
 
     <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
 
+    <PackageReference Include="Testcontainers" Version="3.6.0" />
+    <PackageReference Include="StrongNamer" Version="0.2.5" />  
```

Comment:
grmbl grmbl ihatestrongnaming grmbl grmbl

Reply:
We all do